tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/RetryConditionTest.java

```diff
@@ -396,7 +397,7 @@ private void assertCompleteExceptionally(final CompletionStage<String> future,
                                              final Class<? extends Throwable> exceptionClass,
                                              final String exceptionMessage) {
         try {
-            CompletableFutureHelper.toCompletableFuture(future).get(1000, TimeUnit.MILLISECONDS);
+            CompletableFutureHelper.toCompletableFuture(future).get(TCKConfig.getConfig().getTimeoutInMillis(), TimeUnit.MILLISECONDS);
```

Comment:
Is this call, and the one below, missing a `getTimeoutInMillis()` argument?

Reply:
I see there's an argument-less variant which defaults to 1000, but I'd lean towards being explicit.

Reply:
Yeah, I agree. I think there was the idea that most tests should use the same timeouts, but I think that's a larger, more difficult, thing to accomplish. I've removed the default timeout method.